### PR TITLE
Added Podman in HPC environments article

### PR DESCRIPTION
**Pull request by adrianreber**

### CODE REVIEW

**Review by vrothberg — COMMENTED** [review]
LGTM, thanks a lot, @adrianreber!

@TomSweeneyRedHat @fatherlinux @rhatdan PTAL

File: _posts/2019-09-25-podman-in-hpc.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+$ cat Dockerfile
+FROM registry.fedoraproject.org/fedora:30
+
+RUN dnf -y install openmpi && \
+    dnf clean all
+
+COPY ring /home/ring
+```
+
+After building the container (`podman build`) I pushed the container to the
```
Comment (TomSweeneyRedHat) [inline _posts/2019-09-25-podman-in-hpc.md]: It would be nice to show the actual `podman build` command that you used here.

Comment (TomSweeneyRedHat) [reply _posts/2019-09-25-podman-in-hpc.md]: and the push

File: _posts/2019-09-25-podman-in-hpc.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+   This tells Open MPI to create all its temporary files in `/tmp/podman-mpirun`
+   and not in `/tmp`. If this is not specified Open MPI will create its temporary
+   files in a directory with a host name in it in `/tmp` and if using more than one
+   node this directory will be named differently on other nodes. This requires
+   mounting the complete `/tmp` directory into the container which is a bit more
+   complicated due to not being able to change SELinux labels of `/tmp`.
+
+This are already all parameters necessary for `mpirun` and now the command is
+specified which `mpirun` should start. `podman` in this case.
```
Comment (TomSweeneyRedHat) [inline _posts/2019-09-25-podman-in-hpc.md]: suggest:   "That's if for all the necessary parameters for `mpirun`, now the command is specified that `mpirun` should start;`podman` in this case."

**Review by TomSweeneyRedHat — COMMENTED** [review]
Just a few nits, really awesome blog entry.  Thanks for putting it together @adrianreber !

File: _posts/2019-09-26-podman-in-hpc.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+RUN dnf -y install openmpi && \
+    dnf clean all
+
+COPY ring /home/ring
+```
+
+After building the container (`podman build --tag=mpi-test:31 .`) I pushed the
+container to the [quay.io](https://quay.io) container registry (`podman push
+mpi-test:31 quay.io/adrianreber/mpi-test:31`) and can now pull it like this:
```
Comment (TomSweeneyRedHat) [inline _posts/2019-09-26-podman-in-hpc.md:67]: :+1:   TY!

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Pull request merged**